naresh/src/main/java/com/lftechnology/phpjava/ems/views/EmployeeView.java

```diff
@@ -0,0 +1,373 @@
+package com.lftechnology.phpjava.ems.views;
+
+import com.lftechnology.phpjava.ems.constants.Constant;
+import com.lftechnology.phpjava.ems.controllers.EmployeeController;
+import com.lftechnology.phpjava.ems.entities.Employee;
+import com.lftechnology.phpjava.ems.enums.Role;
+import com.lftechnology.phpjava.ems.utlis.CommonUtility;
+import com.lftechnology.phpjava.ems.utlis.ConsoleWriter;
+import com.lftechnology.phpjava.ems.utlis.Router;
+import com.lftechnology.phpjava.ems.utlis.UserInput;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * EmployeeView
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 16 2016
+ */
+public class EmployeeView implements ViewSignature<Object> {
+    public static CommonUtility commonUtility = new CommonUtility();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static commonUtility" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make commonUtility a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)